src/util.test.ts

```diff
@@ -88,13 +88,15 @@ describe('validateParams should correctly validate user input', () => {
       { path: '/test/param1', value: 'test', secure: false, description: 'valid'},
       { path: 'test/param2', value: 'test', secure: true, description: 'valid'},
       { path: 'test/param3', value: 'test,test2', secure: true , type: 'StringList' as SSMParamTypes, description: 'valid'},
+//      { path: 'test/param4', value: 'test', tier: 'Advanced' as SSMTierTypes, description: 'valid'},
```

Comment:
I think it's best you remove these comments as they clutter the code. Add them back (uncommented) once you get it working